app/views/dashboards/_actions.html.erb

```diff
@@ -9,6 +9,6 @@
   </a>
   <a class="action <%= "active" if params[:action] == "following" %>" href="/dashboard/following">
     <span>FOLLOWING</span>
-    <span>(<%= @user.following_users_count + @user.following_tags_count %>)</span>
+    <span>(<%= @user.following_users_count + @user.following_tags_count + @followed_organizations.count %>)</span>
```

Comment:
I think you meant `followed_organizations_count`, the cache column on `users` and not calling `.count` on `@followed_organizations`?

Reply:
@Zhao-Andy I used the same value as in `following.html.erb` that was originally added in the PR mentioned in my description above.
https://github.com/thepracticaldev/dev.to/blob/8f00c4b883ba1cb9e31390647f64899eefaec4e2/app/views/dashboards/following.html.erb#L48

Reply:
Oh gotcha. That should probably be `@user.following_organizations_count`, too. :sweat_smile:

Reply:
Should I change that too in this PR?

Reply:
@Glennmen I think it would be nice to change it to `@user.following_organizations_count` in both places.